build/flaws.js

```diff
@@ -485,4 +506,21 @@ async function fixFixableFlaws(doc, options, document) {
   }
 }
 
+function getImageminPlugin(fileName) {
+  const extension = path.extname(fileName);
+  if ((extension === ".jpg") | (extension === ".jpeg")) {
+    return imageminMozjpeg();
+  }
+  if (extension === ".png") {
+    return imageminPngquant();
+  }
+  if (extension === ".gif") {
+    return imageminGifsicle();
+  }
+  if (extension === ".svg") {
+    return imageminSvgo();
```

Comment:
We need to be careful with `svgo` as it often strips of accessibility-related attributes. I know we used to call it with `--disable={removeViewBox,removeTitle}` and then also had this in the webpack config: `plugins: [{ removeUnknownsAndDefaults: false }],`

Reply:
Is that true for SVGs used within the content?

Reply:
I honestly have no idea. Perhaps `removeTitle` is not that big of a deal but, `removeViewBox` and `removeUnknownsAndDefaults` would be useful? I assume you can pass those config bits to `imagemin`?

Reply:
I would also just mimic what we already do in other places. To optimized SVGs are also annoying and less accessible.